src/mysql.py

```diff
@@ -52,13 +59,15 @@ def main():
         if row[0] in sysdbs:
             continue
         cur.execute(
-            'SELECT round(DATA_FREE / 1024 / 1024) '
+            'SELECT table_name, round(DATA_FREE / 1024 / 1024), '
+            'round((data_length + index_length), 2) '
             'FROM information_schema.tables '
             'WHERE TABLE_SCHEMA = %s AND DATA_FREE > 0',
```

Comment:
It is terrible that some columns are written lower case some upper case.  MySQL shows anything under "information_chema" all upper case, but they are strangely case in-sensitive.  I would still write the column names upper case, and the table name as "information_schema.TABLES" in here.